codecov.yml

```diff
@@ -15,5 +15,7 @@ coverage:
     # changes tells us if there are unexpected code coverage changes in other files
     # which were not changed by the diff
     changes: false
+  ignore:  # ignore testutils for coverage
+    - "tuf/testutils/*"
```

Comment:
thanks for adding this!  A super non-blocking nit while we're editing the codecov config: did we want to change the badge colors (something like `range: 50...100` might work)?